src/Stratis.Bitcoin.Features.BlockStore/BlockStoreQueue.cs

```diff
@@ -170,7 +170,7 @@ public Task<uint256> GetTrxBlockIdAsync(uint256 trxid)
         /// <inheritdoc/>
         public Task<Block> GetBlockAsync(uint256 blockHash)
         {
-            throw new System.NotImplementedException();
+            return this.blockRepository.GetAsync(blockHash);
```

Comment:
Please use `.ConfigureAwait(false)` where possible.

Reply:
ok

Reply:
in that case there are a few other places that need it : 
line 161, line 167, line 153